### ISSUE TITLE
[UI] Istio - Always show "Labels" rather than require clicking "Show advanced options"

### ISSUE THREAD

**Issue by davidnuzik** [rancher/rancher#21894]
Version: master-head (7/31/19)

**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
- Create or edit a Virtual Service
- Create or edit a Destination Rule

**Result:**
Both have at the very bottom of the page a "Show advanced options" link that just reveals labels.
Can we just have Labels always show on these two pages? It's not ideal in the UI to have this listed twice (yes for to different things, but looks a bit odd). We  think we should just show the Labels pane (just keep it collapsed unless user opens it).
![image](https://user-images.githubusercontent.com/45179589/62252573-12a21300-b3a8-11e9-9e79-231f0338b2f0.png)

So, in short, just show "Labels & Annotations" pane all the time for these two pages -- do not require user to click on "Show advanced options" link to reveal it.

**Comment by davidnuzik** [comment]
Version: master-head (8/5/19)

Now when you create a virtual service or destination rule, "Labels & Annotations" pane is always shown. Rather than requiring click on "Show advanced options" link which has been removed:

![image](https://user-images.githubusercontent.com/45179589/62489848-ad219e00-b77c-11e9-89a2-48127867e9d2.png)
